dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -746,6 +746,7 @@ class PerfTestWithSkSL extends PerfTest {
         'run',
         '--verbose',
         '--verbose-system-logs',
+        '--purge-persistent-cache',
```

Comment:
Do we need this in the other `run` spots in this file (like `_launchApp`)?

Reply:
The `_launchApp` is only used by memory test and `--purge-persistent-cache` shouldn't affect it much.

However, in general I agree that adding `--purge-persistent-cache` would make the test more robust just in case that some strange interactions may happen in the future. Even better, I was hoping to have a `--force-reinstall` flag so other stuff could also be cleared.

For this specific flaky situation, I'd like to land this PR first and check if the flakiness is fixed as expected. After that, we can either add `--purge-persistent-cache` to more places, or add `--force-reinstall`.

Reply:
BTW, for adding more `--purge-persistent-cache` in the future, I'll probably add one flag at a time just to make sure that there's no unexpected performance changes caused by this flag.